tests/test_h2o_benchmarks.py

```diff
@@ -0,0 +1,132 @@
+"""
+h2o-ai benchmark groupby part running on coiled.
+"""
+
+import uuid
+
+import coiled
+import dask.dataframe as dd
+import pandas as pd
+import pytest
+from dask.distributed import Client
+
+from .utils import runtime_software_env
+
+
+@pytest.fixture(scope="module")
+def cluster():
+    with coiled.Cluster(
+        software=runtime_software_env,
+        name="h2o-groupby-benchmark_" + str(uuid.uuid4()),
+        account="dask-engineering",
+        n_workers=4,
+        backend_options={"spot": False},
+    ) as cluster:
```

Comment:
If possible I recommend using Coiled Declarative.  It's significantly more reliable.

Reply:
Being handled separately over in https://github.com/coiled/coiled-runtime/pull/34